tests/unit/source_code/test_dependencies.py

```diff
@@ -94,25 +94,6 @@ def test_dependency_resolver_raises_problem_with_unfound_local_notebook_dependen
     ]
 
 
-def test_dependency_resolver_raises_problem_with_non_constant_local_notebook_dependency(mock_path_lookup):
-    notebook_loader = NotebookLoader()
-    notebook_resolver = NotebookResolver(notebook_loader)
-    pip_resolver = PythonLibraryResolver(Whitelist())
-    dependency_resolver = DependencyResolver(pip_resolver, notebook_resolver, [], mock_path_lookup)
-    maybe = dependency_resolver.build_notebook_dependency_graph(Path('root10.py'))
-    assert list(maybe.problems) == [
-        DependencyProblem(
-            'dependency-not-constant',
-            "Can't check dependency not provided as a constant",
-            Path('root10.py'),
-            2,
-            0,
-            2,
-            35,
-        )
-    ]
```

Comment:
I'm curious about why this was removed. I'm assuming that due to the switch to astroid we can now handle the `root10.py` fixture because it can infer the value?

Should the test instead be updated to assert that this _doesn't_ trigger advice now?
Should the fixture instead refer to a symbol that's not available within the immediate context?

Reply:
We already have a test for checking value inference. I don't think it's reasonable to keep this test.